### DISCUSSION TITLE
Email Script Results

### DISCUSSION THREAD

**Discussion by Diogenezl** [sadieblue/agilityblue.workspace-scripts#30] [Q&A] [answered]
Is it possible to have the results from a scripted emailed to me? I have a script that retrieves volumes missing their ranges, and I want to automate the process by scheduling the script to run and email me the results.

**Comment by bmward80** [comment, marked as answer]
Hi @Diogenezl,

To send emails out of AB, you need to have a task. I would suggest that you create an "Administrative" style project and then create tasks that you would then create and send emails from to achieve this goal. It can also act as a way for the script to log the results.

Check out this discussion post for an example of creating and sending a task email: [https://github.com/sadieblue/agilityblue.workspace-scripts/discussions/10](https://github.com/sadieblue/agilityblue.workspace-scripts/discussions/10)

Let me know if you need any further assistance.

Brandon

**Reply by Diogenezl** [reply]
Hi Brandon,

Thanks for the suggestion. I'll take a look.
